packages/web/src/components/result/ReactiveList.js

```diff
@@ -152,15 +164,24 @@ class ReactiveList extends Component {
 		}
 
 		// called when page is changed
-		if (this.props.pagination && this.state.isLoading) {
-			if (nextProps.onPageChange) {
-				nextProps.onPageChange();
-			} else {
-				window.scrollTo(0, 0);
+		if (this.props.pagination) {
+			if (this.state.isLoading) {
+				if (nextProps.onPageChange) {
+					nextProps.onPageChange({
```

Comment:
We can clean up the callback param here. Pass `currentPage` and `totalPages` as two params rather than an object.